dune

```diff
@@ -16,13 +16,27 @@
  (package multicoretests)
  (deps (alias src/default)))
 
+; The main test alias
+(alias
+ (name testsuite)
+ (package multicoretests)
+ (deps
+  (alias_rec src/runtest)))
+
+; The internal tests alias
+(alias
+ (name internaltests)
+ (package multicoretests)
+ (deps
+  (alias_rec test/runtest)))
+
 ; The aliases to control what is run in CI
 ; It can either be the full test suite, or focus on a single test
 (alias
  (name ci)
  (package multicoretests)
  (deps
-  (alias_rec %{env:DUNE_CI_ALIAS=runtest})))
+  (alias_rec %{env:DUNE_CI_ALIAS=testsuite})))
```

Comment:
Question out of curiosity: does this mean that one can avoid commenting in `; (alias_rec focusedtest)))` below and instead just set/override `dune_ci_alias` to `focusedtest` in `common.yml`? 
This is more a question to aid my understanding... :sweat_smile: 
It's probably preferable to keep the `dune`-file adjustments workflow in such a situation, as one has to adjust in the file anyway to point focus to a particular test... :shrug:

Reply:
That was indeed the idea of defining a `DUNE_CI_ALIAS` variable. But I’m not sure we’ve ever used that possibility, in particular since we have no way to set this variable to run the focused tests in multicoretests-ci. Maybe it should just be dropped (as part of this cleanup?)?

Reply:
Thanks! I suggest we just keep the variable for now.